demo/index.js

```diff
@@ -1,10 +1,14 @@
-import { init, stringifyJson } from './demo.js';
+import { init, stringifyJson, htmlEscape } from './demo.js';
 
 console.log('Demo, API: ', window.revSdk);
 
-/** @type {import("../dist/IVbrickApi").IVbrickBaseEmbed} */
+/** 
+ * A reference to the current Rev SDK VbrickEmbed instance
+ * @type {import("..").IVbrickVideoEmbed & import("..").IVbrickWebcastEmbed}
+ */
 let currentEmbed;
 
+// see demo.js for details - this initializes the configuration form. "embedContent" is called when time to call the SDK
```

Comment:
I'd remove the comment. Like before, the code imports init from demo.js. We don't need to say it twice.